app/routes/search/news-items.js

```diff
@@ -0,0 +1,132 @@
+import Route from '@ember/routing/route';
+import { action } from '@ember/object';
+import { isEmpty } from '@ember/utils';
+import { parse, startOfDay, endOfDay } from 'date-fns';
+import search from 'frontend-kaleidos/utils/mu-search';
+
+export default class SearchNewsItemsRoute extends Route {
+  queryParams = {
+    page: {
+      refreshModel: true,
+      as: 'pagina',
+    },
+    size: {
+      refreshModel: true,
+      as: 'aantal',
+    },
+    sort: {
+      refreshModel: true,
+      as: 'sorteer',
+    },
+  };
+
+  textSearchFields = ['title^3', 'htmlContent'];
```

Comment:
not searching on `subtitle` ?
we transpose the titles on creation between agendaitem and newsItem
agendaitem.title == newsItem.subtitle, agendaitem.shortTitle == newsItem.title

Since title (also named long title) is optional on agendaitems, not searching for subtitles on newsitem will filter a lot of results

Reply:
Hmm, good point. I just applied the ticket literally, where the fields are stated as:

- titel (boost ^3)
- HTML inhoud

In other tickets title & subtitle distinction is explicitly stated, so I assumed it was intentional, but I think you may be right here.

Reply:
I marked a change request, but I did see that you are following the jira ticket.
If we persist on only using `title` and `htmlContent` then this is fine and the request can be ignored.
Wanted to bring it up